feature-libs/product-configurator/rulebased/core/services/configurator-quantity.service.spec.ts

```diff
@@ -0,0 +1,39 @@
+import { TestBed } from '@angular/core/testing';
+import { take } from 'rxjs/operators';
+import { ConfiguratorQuantityService } from './configurator-quantity.service';
+
+describe('ConfiguratorQuantityService', () => {
+  let classUnderTest: ConfiguratorQuantityService;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      providers: [ConfiguratorQuantityService],
+    });
+    classUnderTest = TestBed.inject(ConfiguratorQuantityService);
+  });
+
+  describe('getQuantity', () => {
+    it('should not emit anything until it will be initialized from outside', () => {
```

Comment:
Please work with done callback here also (instead of unsubscribe)

Reply:
Sure!

Reply:
It does not work with done callback

Reply:
I see (because nothing is submitted at all, right?). But then we should re-phrase the test I guess

Reply:
Right!

Reply:
I submitted a solution which is a bit more readable

Reply:
I have already seen